docs/overlay_compute_ngd.md

```diff
@@ -0,0 +1,59 @@
+# overlay compute ngd
+
+_subclass of [overlay](./overlay.md)_
+
+This operation computes the normalized Google distance (co-occurrence frequency) in PubMed abstracts and adds virual edges between qnodes AND/OR knodes.
+
+### examples
+
+- [input](../examples/overlay/messages/07_input_ngd.json), [output](../examples/overlay/messages/08_output_ngd.json)
+
+### input requirements
+
+- Input qgraph contains the two qnodes specified in 'subject_qnode_key' and 'object_qnode_key'
+
+### output guarantees
+
+- At least 1 new edge between qnodes
+
+### allowed changes
+
+- add kedges
+- add qedges
+
```

Comment:
@finnagin @ShaopengLiu1 Will need to include `add bindings to results` under allowed changes

Reply:
"- add bindings to results" has been added to both overlay and overlay_compute_ngd.